app/config/config.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+
+//Site Info
+define('SITE_NAME', 'PetSo');
+
+//App Root (Dynamic Links)
+define('APP_ROOT', dirname(dirname(__FILE__)));
+define('URL_ROOT', 'http://localhost/PetSo');
+define('URL_SUBFOLDER', 'PetSo');
+
+//Database Info
+define('DB_HOST', '13.229.217.85:3306');
+define('DB_USER', 'petsodev');
+define('DB_PASS', "Azma123&*^");
+define('DB_NAME', 'petso');
+
```

Comment:
Please remove the config folder and push again. (Otherwise, the DB credentials are publicly visible).